src/main/java/com/rarchives/ripme/ripper/RipperInterface.java

```diff
@@ -9,10 +9,17 @@
  * Oh well, here's to encapsulation and abstraction! (raises glass)
  */
 public interface RipperInterface {
-    public void rip() throws IOException;
-    public boolean canRip(URL url);
-    public URL sanitizeURL(URL url) throws MalformedURLException;
-    public void setWorkingDir(URL url) throws IOException;
-    public String getHost();
-    public String getGID(URL url) throws MalformedURLException;
-}
+
+    void rip() throws IOException;
+
+    boolean canRip(URL url);
```

Comment:
There's no benefit to all the blank lines here. I think it's more readable in compressed form.